### PR TITLE
Audit fix m2 s11

### PR DESCRIPTION
**Pull request by dxganta**

### CODE REVIEW

File: contracts/strategy/liquity/LiquityStrategy.sol [hunk 1]
```diff
@@ -251,6 +254,10 @@ contract LiquityStrategy is
             revert StrategyNothingToReinvest();
         }
 
+        if (balance < _amountOutMin) {
```
Comment (brkomir) [inline contracts/strategy/liquity/LiquityStrategy.sol:257]: shouldn't this be `<=` ?

**Comment by brkomir** [comment]
Please resolve this merge conflict

**Review by brkomir — APPROVED** [review]
Please target the main branch after[ this PR](https://github.com/lindy-labs/sc_solidity-contracts/pull/202) is merged to maintain a clear history of commits on the main

**Comment by dxganta** [comment]
Too many merge conflicts, so I created a new one and closing this one.

**Pull request closed without merging**